specification/machinelearningservices/resource-manager/readme.go.md

```diff
@@ -24,6 +25,15 @@ batch:
   - tag: package-2020-02-18-preview
   - tag: package-2018-03-preview
 ```
+## Tag: package-2021-07-01 and go
+
+These settings apply only when `--tag=package-2021-07-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag)=='package-2021-07-01' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-07-01/$(namespace)
```

Comment:
Since this tag only contains `stable` swaggers, we need to remove the preview segment in this output-folder path
```suggestion
output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2021-07-01/$(namespace)
```